src/state/opamSysPoll.ml

```diff
@@ -158,3 +158,6 @@ let variables =
       "os-version", os_version_lazy;
       "os-family", os_family_lazy;
     ]
+
+let cores_lazy = lazy (OpamSystem.cpu_count ())
+let cores () = Lazy.force cores_lazy
```

Comment:
I prefer to make `cores_lazy` local in this case:
```
let cores =
  let cores_lazy = ... in
  fun () -> Lazy.force ...
```
(but this is fine; the idea is just to make it clear that cores_lazy won't be otherwise used)

Reply:
It is to have the same _pattern_ as other lazy values.